presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -407,6 +410,10 @@ public void renameSchema(ConnectorSession session, String source, String target)
     @Override
     public void createTable(ConnectorSession session, ConnectorTableMetadata tableMetadata)
     {
+        if (!createNonManagedTablesEnabled) {
```

Comment:
This disallows all create table statements, which isn't quite right -- you can create tables that are managed tables too.